### ISSUE TITLE
Converted all day events "no attribute tzinfo" error writing to ical

### ISSUE THREAD

**Issue by Nycrea** [niccokunzmann/python-recurring-ical-events#7]
When converting all day events with recurring_ical_events, calling `to_ical()` on the resulting calendar triggers the error ```AttributeError: 'datetime.date' object has no attribute 'tzinfo'```

I converted the calendar as shown in the readme, then add the event to a new `icalendar.Calendar` object with `add_component()` and call `to_ical()` on the resulting Calendar object. 

Example input icalendar
```
BEGIN:VCALENDAR
BEGIN:VEVENT
SUMMARY:redacted
DTSTART;VALUE=DATE:20190604
DTEND;VALUE=DATE:20190701
DTSTAMP:20190408T073229Z
UID:00000000-0000-0000-0000-000000000000
SEQUENCE:0
ATTENDEE;CN="redacted";PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:redacted@example.com
CLASS:PUBLIC
DESCRIPTION:redacted
LAST-MODIFIED:20190408T073229Z
ORGANIZER;CN="redacted":mailto:redacted@example.com
STATUS:CONFIRMED
TRANSP:OPAQUE
X-ALT-DESC;FMTTYPE=text/html:<html></html>
X-MICROSOFT-CDO-ALLDAYEVENT:TRUE
X-MICROSOFT-CDO-INTENDEDSTATUS:OOF
BEGIN:VALARM
ACTION:DISPLAY
DESCRIPTION:Reminder
TRIGGER;RELATED=START:-PT10M
END:VALARM
END:VEVENT
END:VCALENDAR
```

Full stacktrace
```
Traceback (most recent call last):
  File "parse_ical.py", line 141, in <module>
    main(sys.argv[1:])
  File "parse_ical.py", line 81, in main
    print((test.to_ical()))
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/cal.py", line 426, in to_ical
    content_lines = self.content_lines(sorted=sorted)
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/cal.py", line 415, in content_lines
    cl = self.content_line(name, value, sorted=sorted)
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/cal.py", line 408, in content_line
    return Contentline.from_parts(name, params, value, sorted=sorted)
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/parser.py", line 306, in from_parts
    values = values.to_ical()
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/prop.py", line 395, in to_ical
    tzid = tzid_from_dt(dt)
  File "/home/user/.local/lib/python2.7/site-packages/icalendar/parser.py", line 54, in tzid_from_dt
    if hasattr(dt.tzinfo, 'zone'):
AttributeError: 'datetime.date' object has no attribute 'tzinfo'
```

**Comment by niccokunzmann** [comment]
Hi, thanks for getting in contact!
Indeed, I did not test any ability to convert events to ical again after they went through this module.
Since this library is developed driven by tests, it would be quite easy to create tests for your use-case to ensure future compatibility.
e.g. look at this pull request: https://github.com/niccokunzmann/python-recurring-ical-events/pull/5/files

- it adds the calendar provided
- adds some tests
- modifies the library

What would you need to take part in improving this library with joy?

**Comment by niccokunzmann** [comment]
I am guessing but the fix is about two lines long:
https://github.com/niccokunzmann/python-recurring-ical-events/blob/39be2121fc4e2a5875a28ef275ffcfc2ebe18544/recurring_ical_events.py#L206

**Comment by niccokunzmann** [comment]
This fix is included since version 0.1.8b.
